### ISSUE TITLE
Focus placed should be a core requirement

### ISSUE THREAD

**Issue by ajsaucier** [w3c/wcag3#616]
"Focus placed" and "Focus retained" seem to be very similar, and both are currently a supplemental requirement. I think it is worth discussing whether they should be a core requirement. I have spoken with screen reader users who have seen it as a barrier when they are in the middle of completing a form and using a date picker, for example, only to have their focus lost when the date picker is closed and sent back to the `<body>` element. Is part of the reason for it being a supplemental requirement that browsers try to return focus to a logical place on their own?
